pom.xml

```diff
@@ -51,52 +54,31 @@
     <dependencies>
         <dependency>
             <groupId>com.github.docker-java</groupId>
-            <artifactId>docker-java</artifactId>
-            <version>3.0.13</version>
+            <artifactId>docker-java-api</artifactId>
+            <version>3.2.0-rc4</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-core</artifactId>
+            <version>3.2.0-rc4</version>
             <exclusions>
-                <!-- use the version supplied by jackson2-api -->
```

Comment:
This is removing a lot of exclusions here, and that makes me nervous...
We don't want this plugin to end up dragging in any more 3rd-party libraries than we absolutely need.
However, I don't know enough about the syntax and/or reasons for having these here to start with to be able to be _sure_ we don't need such exclusions anymore ... or to give any more detailed a review than asking "are you sure?"